CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1523,6 +1529,66 @@ public IWebBrowser WebBrowser
 
         #endregion WebBrowser dependency property
 
+        #region IsAudioMuted dependency property
+
+        /// <summary>
+        /// Whether the browser audio is muted
+        /// </summary>
+        /// <value>The IsAudioMuted value.</value>
+        /// <remarks>In the WPF control, this property is implemented as a Dependency Property and fully supports data
+        /// binding.</remarks>
+        public bool IsAudioMuted
```

Comment:
This doesn't actually represent if the browser is muted, so I'm reluctant to add this, specifically as it's not actually required to implement the command. Using IBrowserHost.IsAudioMuted to toggle would give a consistent behaviour.